build.gradle

```diff
@@ -32,29 +14,30 @@ apply plugin: 'idea'
 apply plugin: 'com.github.johnrengelman.shadow'
 
 group = 'com.airbnb'
-description = 'StatsD Support for Kafka Metrics'
```

Comment:
Yeah, change it to StatsD Support for Kafka Metrics (kafka version: 0.8.1 and 0.8.2)